.aspell.en.pws

```diff
@@ -59,3 +59,5 @@ CPUs
 MiB
 RBAC
 YAML
+de
```

Comment:
I'm not a fan of this, I think it would be better to reword so we don't add an ignore for "de" and "facto" on their own.